app/jobs/add_metadata_job.rb

```diff
@@ -1,19 +1,23 @@
 class AddMetadataJob < ApplicationJob
   queue_as :default
   require 'csv'
-  def perform(csv_data,account_id,asset_id)
+  def perform(csv_data,account_id)
     CSV.parse(File.read(csv_data), headers: true) do |row|
-	    Media.create(
-		asset_id: asset_id,
-		title: row[0],
-		duration: row[1],
-		file_location: row[2],
-		recorded_time: row[3],
-		account_id: account_id
-	)
-	  
-    end
-    
-end
+	   asset_id = ('0'..'z').to_a.shuffle.first(8).join
+	   @media = Media.new(
```

Comment:
Please read about factory_bot and its advantage over using model objects directly